css-three-column-layout/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Three Column Layout</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="fullColumnTitle">
+          <h2>
+            Designed for life science. Optimized for SaMD and Digital Health.
+          </h2>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-one-third">
+          <div class="circle"></div>
```

Comment:
Instead of having the `circle` be a separate empty div, try putting the `img` inside of the `.circle` class element, and use flex properties to align/justify the `img` in the center of the circle wrapper div.